### DISCUSSION TITLE
I want to know the permission or power these individual roles have.

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Lern/Community#23] [Documentation]
Please provide the detailed permissions of the following roles:
- Public
- Content_Creator
- Content_Reviewer
- Course_Mentor
- Org_Admin
- Org_Management
- Org_Moderator 
@reshmi-nair @amit-tarento

**Comment by amitec006** [comment]
@sachinb80 Please find the details about role permission/access here https://github.com/project-sunbird/sunbird-community/discussions/792

**Reply by sachinb80** [reply]
Thanks, Amit
